atomics/T1218/T1218.yaml

```diff
@@ -401,3 +401,19 @@ atomic_tests:
       taskkill -f -im calc.exe
       taskkill -f -im win32calc.exe
     name: powershell
+- name: System Binary Proxy Execution - Wlrmdr Lolbin
+  auto_generated_guid: 
+  description: Use wlrmdr(Windows Logon Reminder executable) as a proxy binary to evade defensive countermeasures
+  supported_platforms:
+  - windows
+  input_arguments:
+    payload_path:
+      description: Path to the executable
+      type: String
+      default: C:\Windows\System32\calc.exe
+  executor:
+    command: wlrmdr.exe -s 3600 -f 0 -t _ -m _ -a 11 -u "#{payload_path}"
+    cleanup_command: 
```

Comment:
Can you remove the empty cleanup_command as well ?

Reply:
@cyberbuff , Apologies for the delay. As requested, I've removed the  empty cleanup_command from the atomics and updated the commit.